.eslintrc

```diff
@@ -87,7 +87,6 @@
     "no-loop-func": 2,               // http://eslint.org/docs/rules/no-loop-func
     "no-multi-str": 2,               // http://eslint.org/docs/rules/no-multi-str
     "no-native-reassign": 2,         // http://eslint.org/docs/rules/no-native-reassign
-    "no-new": 2,                     // http://eslint.org/docs/rules/no-new
```

Comment:
There's a test that needs to create `new Proxy(...)` to test that it wires up callbacks properly.
In that scenario, I don't think it's weird that `new Proxy(...)` has side-effects, so I'm not sure that this lint is benefiting the project :thinking: